.github/workflows/tpch.yml

```diff
@@ -33,11 +34,6 @@ on:
         default: true
         type: boolean
 
-concurrency:
-  # Only allow a single run at a time to rate limiting
-  group: tpch
-  cancel-in-progress: true
```

Comment:
I think we should have some kind of control but maybe a little more granular. I'm fine either way about the job as long as we're a little careful

Reply:
I figured you had some reason to remove it for now so I stuck with that, I'm happy to iterate on this.